lib/api/messages.js

```diff
@@ -1655,6 +1662,10 @@ module.exports = (db, server, messageHandler, userHandler, storageHandler, setti
             } else if (attachmentData.transferEncoding === 'quoted-printable') {
                 attachmentStream.pipe(new libqp.Decoder()).pipe(res);
             } else {
+                if (!/ascii|utf[-_]?8/.test(attachmentCharset.toLowerCase()) && result.value.sendAsString) {
```

Comment:
I you are already using a regex, then it is better to use the case-insensitive flag instead of `toLowerCase()`

```js
!/ascii|utf[-_]?8/i.test(attachmentCharset)
```